requirements.txt

```diff
@@ -1,3 +1,4 @@
 selenium == 3.0.0b2
```

Comment:
FoxPuppet will be pip installable from PyPI, not just a library. Otherwise users would have to clone the repository in order to use it.